### DISCUSSION TITLE
I see you did a previewer like I suggested

### DISCUSSION THREAD

**Discussion by davidbuckleyni** [QuestPDF/QuestPDF#274] [General]
Congrats on the previewer looks well so it does I remember us having a discussion about it on reddit.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
